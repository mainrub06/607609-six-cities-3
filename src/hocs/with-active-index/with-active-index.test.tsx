import React from "react";
import renderer from "react-test-renderer";
import withActiveIndex from "./with-active-index";

const MockComponent = ({ children }: any) => <div>{children}</div>;
const MockComponentWrapped = withActiveIndex(MockComponent);

it(`Render snapshot HOC <withActiveIndex/> with mock Component`, () => {
  const tree = renderer
    .create(<MockComponentWrapped activeItemIndex={0} />, {
      createNodeMock() {
        return {};
      },
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
