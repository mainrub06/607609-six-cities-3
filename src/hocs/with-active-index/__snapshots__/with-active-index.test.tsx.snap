// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render snapshot HOC <withActiveIndex/> with mock Component 1`] = `<div />`;
